sdk/pom.xml

```diff
@@ -44,6 +44,11 @@
             <artifactId>gson</artifactId>
             <version>2.11.0</version>
         </dependency>
+        <dependency>
+            <groupId>com.vdurmont</groupId>
+            <artifactId>semver4j</artifactId>
```

Comment:
I don't think you need to add a dep to do `TDF_VERSION + "+" + escape(buildMetadata)` (and the reverse)